### ISSUE TITLE
add-path deprecation requires actions/core version bump

### ISSUE THREAD

**Issue by scttl** [Wizhi/setup-buf#3]
Now that github actions has deprecated `add-path` using the setup-buf action fails as follows:
```
Error: Unable to process command '::add-path::/opt/hostedtoolcache/buf/0.21.0/x86_64' successfully.
Error: The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

I can work around by setting the environment variable listed above to true, but it would be good to accept #2 to fully address this issue (the underlying actions/core dependency uses the deprecated add-path call but it has been remedied in 1.2.6)

**Comment by TanyaDolgopolova** [comment]
I have the same problem. Did you resolve it?

**Comment by wizhi** [comment]
@scttl @TanyaDolgopolova 

Hello. :)

This has been fixed as of dcdcf2efebe862fd4b75f98e512737cc937666d6. 
I've released [`1.1.1`][0] tag, which includes the update in question, along with another dependency update.

I've also updated the [`v1`][1] release tag reference (it now points to [`1.1.1`][0]), so this should result in a transparent change for you.

[0]: https://github.com/Wizhi/setup-buf/releases/tag/1.1.1
[1]: https://github.com/Wizhi/setup-buf/releases/tag/v1
